gax-httpjson/src/test/java/com/google/api/gax/httpjson/ProtoRestSerializerTest.java

```diff
@@ -53,7 +60,14 @@ public class ProtoRestSerializerTest {
 
   @Before
   public void setUp() {
-    requestSerializer = ProtoRestSerializer.create();
+    // tests with Any type messages require corresponding descriptors in the type registry
+    requestSerializer =
+        ProtoRestSerializer.create(
+            TypeRegistry.newBuilder()
```

Comment:
Can you check if these descriptors are added to ProtoRestSerializer on generating client libraries? If not, we should added the well knowntypes to the registry as well.